src/styles/BookInfo.css

```diff
@@ -0,0 +1,51 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;300;400;500;600;700;800;900&display=swap');
+
```

Comment:
- [ ] kindly consider sticking to the given design as much as possible to make your design look exactly as expected check the image below for more clarification
- your design 
<img width="1397" alt="Screen Shot 2023-06-23 at 13 24 49" src="https://github.com/Megagig/Bookstore/assets/60146030/3e21e94d-fc2c-4f8f-a6eb-64970a4faf33">